proto/rill/runtime/v1/resources.proto

```diff
@@ -187,6 +187,7 @@ message MetricsViewSpec {
     string lookup_table = 8;
     string lookup_key_column = 9;
     string lookup_value_column = 10;
+    optional string lookup_default_value = 11; // optional because need to distinguish between not set and empty value
```

Comment:
Can it be an expression instead, like `lookup_default_expression`? Then it could work for non-string values and wouldn't need `optional`